jenkins/ci_build_win_desktop.ps1

```diff
@@ -85,16 +90,23 @@ function Run-UnitTest() {
     Pop-Location
 }
 
-Remove-Item -Recurse -Force -ErrorAction Ignore "${env:WORKSPACE}\build_x64\libcblite-${Version}"
+$Build_Dir = "build_${Architecture}"
+Remove-Item -Recurse -Force -ErrorAction Ignore "${env:WORKSPACE}\${Build_Dir}\libcblite-${Version}"
 New-Item -Type Junction -Target ${env:WORKSPACE}/couchbase-lite-c-ee/couchbase-lite-core-EE -Path ${env:WORKSPACE}/couchbase-lite-c/vendor/couchbase-lite-core-EE
-Build "${env:WORKSPACE}\build_x64"
+Build "${env:WORKSPACE}\${Build_Dir}"
 if("${Edition}" -eq "enterprise") {
-    Run-UnitTest "${env:WORKSPACE}\build_x64"
+    Run-UnitTest "${env:WORKSPACE}\${Build_Dir}"
 }
 
-Make-Package "${env:WORKSPACE}\build_x64" "couchbase-lite-c-$Edition-$Version-$BuildNum-windows-x86_64.zip"
+$Package_Arch = ${Architecture}.ToLowerInvariant()
+if ("${Package_Arch}" -eq "x64") {
```

Comment:
Might want to also lower case the arm64 for package name, but that's a stylistic thing.

Reply:
It already does. See line 101.